### ISSUE TITLE
Change 'too many datasets' error message text colour

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#1106]
As a result of recent high contrast changes, it has been noticed that the colour contrast in the 'too many datasets' error message is no longer meets 'AA' compliance. 

Recommendation: Change error text to black in high contrast mode.
![Screenshot (150)](https://user-images.githubusercontent.com/60919820/108985109-91e0f100-7688-11eb-990f-017848e0810d.png)
